tests/smoke-tests/src/main/java/com/techorgx/api/payload/User.java

```diff
@@ -0,0 +1,53 @@
+package com.techorgx.api.payload;
+
+public class User {
+    String city;
```

Comment:
Can we make these fields private if we have getters and setters for them.

Reply:
Done